Makefile

```diff
@@ -9,8 +9,15 @@ check_nasm:
 		fi \
 	fi
 
+SPEC_VERSION ?= v1.5.0-alpha.8
+
 # Update deps target to include nasm check
 deps: check_nasm
 	@echo "Installing dependencies"
 	@git submodule update --init --recursive
 	@cd bls && make -f Makefile.onelib ETH_CFLAGS=-DBLS_ETH LIB_DIR=lib
+
+	mkdir -p consensus-spec-tests
```

Comment:
Use a separate the task would be better such as if someone want to build it in local but don't want to do spec tests.

Reply:
done